CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing 
+Thanks for helping us make the HPE Design System better! You came to the right place to start your contribution.
+
+## Project Structure
+The HPE Design System is a monorepo built from two modules.
+See the [README](https://github.com/hpe-design/design-system#welcome-to-hpe-design-system)
+for a break down of the Design System structure.
+
+## How to Contribute 
+Code, Code, Code...
+
+## Submitting Pull Requests 
+To make a pull request you will need a GitHub account. For help, see
+GitHub’s documentation on [forking] and [pull requests].
+1. Fork the design-system repository
```

Comment:
Do we want folks to fork or clone the repo? Right now, this is written as both.

Reply:
They would have to do both Im assuming. Like grommet you have to fork then clone the project since your cant directly push a PR and have to do this through the fork